scalableminds/webknossos
# Add API requests for showing active tasks and transferring them

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- call `/api/projects/sampleProject/usersWithActiveTasks`
- call `/api/projects/sampleProject/transferActiveTasks` with a valid `userId` as body and see if this user now has all active tasks

### Issues:
- fixes #2862 

------
